src/app/pages/client-review/review.component.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { NgxSpinnerModule } from 'ngx-spinner';
+import { ReactiveFormsModule, FormsModule, NgForm } from '@angular/forms';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ToastrModule } from 'ngx-toastr';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { throwError, of, Subject } from 'rxjs';
+import { Router, ActivatedRoute } from '@angular/router';
+
+import { ReviewComponent } from './review.component';
+
+class RouterStub {
```

Comment:
Hey @Niyitangasam , I think we can use the spy that is inside the `helpers/spies.ts` file